### PR TITLE
New terms from current customer PoC

### PR DESCRIPTION
**Pull request by pmcb55**
# New feature description

Cherry-picked some new generic terms from Schema.org, and added a couple of generically useful terms to the Inrupt Common vocab (for concepts that don't seem to exist in Schema.org).

# Checklist

- [X] All acceptance criteria are met.
- [X] Relevant documentation, if any, has been written/updated.
- [X] The changelog has been updated, if applicable.
- [X] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

File: inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl [hunk 1]
```diff
@@ -54,10 +62,12 @@ inrupt_common:acronymExpandedForm a rdfs:Property ;
...
 #
 inrupt_common:errFailedToProcessIncomingRdf a skosxl:Label ;
-	skos:definition "Failed to process incoming message as RDF"@en .
+	skos:definition "Failed to process incoming message as RDF."@en ;
+    skos:definition "No se pudo procesar el mensaje entrante como RDF."@es .
 
 inrupt_common:msgRequestValidatedAsRdf a skosxl:Label ;
-	skos:definition "Incoming request validated as RDF (it has [{{0}}] triples!)"@en .
+    skos:definition "Incoming request validated as RDF (it has [{{0}}] triples!)."@en ;
+	skos:definition "Solicitud entrante validada como RDF (¡tiene [{{0}}] triples!)."@es .
```
Comment (andydavison) [inline inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl:70]: some tab/space inconsistent indentation here

File: inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl [hunk 2]
```diff
@@ -19,8 +19,10 @@ prefix inrupt_common: <https://inrupt.com/vocab/common#>
 inrupt_common: a owl:Ontology, inrupt_gen:Ontology ;
 	owl:versionInfo "0.0.1" ;
 	owl:versionIRI <https://inrupt.com/vocab/common/0.0.1> ;
- 	dcterms:title "Inrupt common vocabulary"@en ;
+    dcterms:title "Inrupt common vocabulary"@en ;
+    dcterms:title "Inrupt vocabulario común"@es ;
```
Comment (andydavison) [inline inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl:23]: mismatched indentation with stuff around this

**Review by andydavison — APPROVED** [review]
just some small indent/alignment things to check (mismatch between spaces and tabs I think), otherwise LGTM

File: inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl (hunk 2 above)
Comment (pmcb55) [reply inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl:23]: Fixed: f3e6d5a

File: inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl (hunk 1 above)
Comment (pmcb55) [reply inrupt-rdf/Core/CopyOfVocab/inrupt-common.ttl]: Fixed: f3e6d5a

**Pull request merged**